sympy/sets/fancysets.py

```diff
@@ -1338,6 +1345,8 @@ def _contains(self, other):
         isTuple = isinstance(other, Tuple)
         if isTuple and len(other) != 2:
             raise ValueError('expecting Tuple of length 2')
+        if other is S.ComplexInfinity:
+            return False
```

Comment:
`{Re[ComplexInfinity], Im[ComplexInfinity]}` both indeterminate. in this link : http://reference.wolfram.com/language/ref/ComplexInfinity.html